package.json

```diff
@@ -91,6 +89,8 @@
     "webpack-stream": "^3.2.0"
   },
   "dependencies": {
+    "@storybook/addon-options": "^3.1.2",
+    "@storybook/react": "^3.1.3",
```

Comment:
@orta I’m wondering if it was intentional to add these as runtime dependencies? I’m going to assume not and will move them to dev deps.